index.d.ts

```diff
@@ -90,6 +90,7 @@ interface ClientOptions {
   generateRequestId?: generateRequestIdFn;
   name?: string;
   auth?: BasicAuth | ApiKeyAuth;
+  context?: Record<string, any>;
```

Comment:
prefer to use `unknown`. it will enforce type casting before reading https://mariusschulz.com/blog/the-unknown-type-in-typescript